js/popupView.js

```diff
@@ -0,0 +1,145 @@
+define(function(require) {
+
+    var Adapt = require('coreJS/adapt');
+
+    var PopupView = Backbone.View.extend({
+
+        currentIndex: 0,
+        isOpen:false,
+
+        events: {
+            'click .back': 'previousHotGraphic',
+            'click .next': 'nextHotGraphic'
+        },
+
+        initialize:function(options) {
+            this.currentIndex = options.index || 0;
+            this.isOpen = false;
+            this.render();
+
+            this.listenToOnce(Adapt, 'popup:opened', this.onPopupOpened);
+            this.listenToOnce(Adapt, 'popup:closed', this.onPopupClosed);
+        },
+
+        render:function() {
+            var template = Handlebars.templates['hotgraphicPopup'];
```

Comment:
This isn't going to need to support IE8/9/10 so I would encourage you to drop `var` and use `const` and `let` instead